test/module/irohad/consensus/yac/yac_synchronization_test.cpp

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "module/irohad/consensus/yac/yac_fixture.hpp"
+
+#include "common/hexutils.hpp"
+
+using namespace iroha::consensus::yac;
+
+using ::testing::_;
+using ::testing::Return;
+
+/**
+ * The class helps to create fake network for unit testing of consensus
+ */
+class NetworkUtil {
+ public:
+  /// creates fake network of number_of_peers size
+  NetworkUtil(size_t number_of_peers) {
+    for (size_t i = 0; i < number_of_peers; ++i) {
+      peers_.push_back(makePeer(std::to_string(i)));
+    }
+    order_ = ClusterOrdering::create(peers_);
+  }
+
+  auto createHash(const iroha::consensus::Round &r,
+                  const std::string &block_hash = "default_block",
+                  const std::string &proposal_hash = "default_proposal") const {
+    return YacHash(r, proposal_hash, block_hash);
+  }
+
+  auto createVote(size_t from, const YacHash &yac_hash) const {
+    return iroha::consensus::yac::createVote(
+        yac_hash,
+        *iroha::hexstringToBytestring(peers_.at(from)->pubkey().hex()));
+  }
+  /// create votes of peers by their number
+  auto createVotes(
+      const std::vector<size_t> &peers,
+      const iroha::consensus::Round &r,
+      const std::string &block_hash = "default_block",
+      const std::string &proposal_hash = "default_proposal") const {
```

Comment:
If you rely on these literals to be equal, it is better to store them in a single place, like a global const.
https://github.com/hyperledger/iroha/blob/d0a9e924c4498b89c70f8560587d5fa95429254f/test/module/irohad/consensus/yac/yac_synchronization_test.cpp#L29-L30
https://github.com/hyperledger/iroha/blob/d0a9e924c4498b89c70f8560587d5fa95429254f/test/module/irohad/consensus/yac/yac_synchronization_test.cpp#L43-L44

Reply:
Values by itself don't matter. If a user wants to check some specific value it will be done explicitly by a test.